hadoop-ozone/dist/src/main/smoketest/debug/ozone-debug-tests.robot

```diff
@@ -0,0 +1,60 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+*** Settings ***
+Documentation       Test ozone Debug CLI
+Library             OperatingSystem
+Resource            ../commonlib.robot
+Resource            ozone-debug.robot
+Test Timeout        5 minute
+Suite Setup         Write keys
+*** Variables ***
+${VOLUME}           vol1
+${BUCKET}           bucket1
+${DEBUGKEY}         debugKey
+${TESTFILE}         testfile
+
+*** Keywords ***
+Write keys
+    Execute             ozone sh volume create o3://om/${VOLUME} --space-quota 100TB --namespace-quota 100
+    Execute             ozone sh bucket create o3://om/${VOLUME}/${BUCKET}
+    Execute             ozone sh key put o3://om/${VOLUME}/${BUCKET}/${DEBUGKEY} /opt/hadoop/NOTICE.txt
+    Execute             dd if=/dev/urandom of=testfile bs=100000 count=15
+    Execute             ozone sh key put o3://om/${VOLUME}/${BUCKET}/${TESTFILE} testfile
+
+*** Test Cases ***
+Test ozone debug chunkinfo
+    ${result} =     Execute             ozone debug chunkinfo o3://om/${VOLUME}/${BUCKET}/${DEBUGKEY} | jq -r '.KeyLocations[0][0].Locations'
+                    Should contain      ${result}       files
+    ${result} =     Execute             ozone debug chunkinfo o3://om/${VOLUME}/${BUCKET}/${DEBUGKEY} | jq -r '.KeyLocations[0][0].Locations.files[0]'
+                    File Should Exist   ${result}
+
+Test ozone debug read-replicas
+    Execute                             ozone debug read-replicas o3://om/${VOLUME}/${BUCKET}/${TESTFILE}
```

Comment:
First few checks/lines  are common in all three tests, Can we move to a common function?

Reply:
Thanks, this is a good idea. I added a keyword in the ozone-debug to execute the read-replicas there and do the common checks.